### PR TITLE
Chore/analytics

### PR DESCRIPTION
**Pull request by jennytellne-cygni**

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -5,7 +5,7 @@ import PackageDescription
 
 let package = Package(
     name: "NFCPassportReader",
-    platforms: [.iOS(.v13)],
+    platforms: [.iOS(.v13), .macOS(.v10_12)],
```
Comment (sharizard) [inline Package.swift:8]: Trengs .macOs også?

File: Package.swift [hunk 2]
```diff
@@ -26,7 +29,9 @@ let package = Package(
             name: "NFCPassportReader",
             dependencies: [
                 "OpenSSL",
-                .product(name: "Logging", package: "swift-log")
+                .product(name: "Logging", package: "swift-log"),
```
Comment (sharizard) [inline Package.swift:32]: Jeg antar at vi klarer å logge på modul-nivå? :D

File: Package.swift (hunk 1 above)
Comment (jennytellne-cygni) [reply Package.swift:8]: Nope, but swift packages seems to be designed cross platform and Firebase required a minimun macOS version of 10.12 so I set that to make the build-and-test pipeline pass. Doesn't seem to be a way to exclude a platform.

File: Package.swift (hunk 2 above)
Comment (jennytellne-cygni) [reply Package.swift:32]: Yes! 😃

**Review by sharizard — APPROVED** [review]

**Pull request merged**